src/css/unity-custom.scss

```diff
@@ -752,6 +752,39 @@ html[data-theme='dark'] .node polygon {
   --ifm-link-color: #757575;
 }
 
+.footer__legal ul {
+  margin: 1rem 0;
+  padding: 0;
+  display: flex;
+  flex-flow: row nowrap;
+  gap: 1rem;
+  list-style-type: none;
+
+  a {
+    color: var(--ifm-footer-title-color);
+  }
+}
+
+.footer__legal-onetrusttick {
+  margin-right: 0.5rem;
+}
+
+// override the OneTrust styling for the link-button
```

Comment:
Note: I dislike using `!important` here however it was necessary since I don't think we have control over the order of the Styles in the header and this is needed to override the OneTrust styles.